### DISCUSSION TITLE
[18.06]移植LEAN的ZTE E8820S支持到18.06上的问题

### DISCUSSION THREAD

**Discussion by wsvg14** [immortalwrt/immortalwrt#487] [General]
[https://github.com/wsvg14/immortalwrt/blob/openwrt-18.06/target/linux/ramips/dts/mt7621_zte_e8820s.dts](url)
[https://github.com/wsvg14/immortalwrt/blob/openwrt-18.06/target/linux/ramips/image/mt7621.mk](url)
[https://github.com/wsvg14/immortalwrt/blob/openwrt-18.06/target/linux/ramips/mt7621/base-files/etc/board.d/01_leds](url)
[https://github.com/wsvg14/immortalwrt/blob/openwrt-18.06/target/linux/ramips/mt7621/base-files/etc/board.d/02_network](url)

以上是我目前使用在18.06编译中的文件，从大雕那边的5.X内核文件修改过来尝试适配。
但是现在的情况是，如果用大雕那边默认的7621内核软件配置,那么无线显示会变成双7615，而非8820S的MT7603E+MT7612E，从而根本没法启动wifi。
如果使用b70的内核软件配置，则无线选项和相关文件直接消失，同样不可用。
有线部分暂时正常使用中。

因此尝试发个issue来寻求下帮助，~~顺带看看能不能加设备支持~~（从天灵佬那边过来的）

**Comment by 1715173329** [comment]
mtwifi on 18.06 is for k2p only
我建议你用 18.06-k5.4 或者 master 分支
fyi https://github.com/siwind/openwrt/tree/master/target/linux/ramips
